### PR TITLE
docs: add back-pressure and cancellation blog

### PR DESCRIPTION
**Pull request by gyxlucy**
Add a new blogpost.

### CODE REVIEW

File: website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+authors: 
+  - wwayne
+  - icycodes
+  - gyxlucy
+
+tags: [tech design]
+---
+# HTTP Streaming Back-Pressure and Cancellation in Code Completion
+
+![back-pressure](./intro.png)
```
Comment (wsxiaoys) [inline website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md:11]:
```suggestion
![Introduction](./intro.png)
```

Alt string is mostly used for search engine understanding the content of image, so it should be semantic relevant to image.

Comment (wsxiaoys) [reply website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md:11]: Please also fix other cases.

File: website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+  const serverStream = new ReadableStream({
+    async pull(controller) {
+      const { value, done } = await reader.read();
+      if (done) {
+        controller.close();
+      } else {
+        controller.enqueue(value);
+      }
+    }
+  });
```
Comment (icycodes) [inline website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md:156]: The code sample appears to have mixed `\t` and space indentation, as well as variation in the use of semicolons.  
It is recommand use Prettier to format the code sample.

File: website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+    isConnectionClosed = true
+  })
+
+	// 2. Get the methods from LLM
+	const { cancel, text } = llmInference(prompt)
+
+  // 3. Create a ReadableStream
+  const serverStream = new ReadableStream({
+    async pull(controller) {
+      const { value, done } = await reader.read();
```
Comment (icycodes) [inline website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md:230]: This code sample appears to be confusing, there is no `reader` in the context.

Comment (gyxlucy) [reply website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md:230]: @wwayne Could you please take a look at the code example? Thanks

Comment (wwayne) [reply website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md:230]: Thanks, I sent a PR for the suggestions above https://github.com/TabbyML/tabby/pull/496

File: website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md (hunk 2 above)
Comment (wwayne) [reply website/blog/2023-09-26-http-streaming-back-pressure-and-cancellation/index.md:156]: Thanks, didn't notice that when I was writing these on Notion at first

**Pull request merged**